docs/how-to/using_helm_kustomize_manage_oam.md

```diff
@@ -0,0 +1,196 @@
+# Using Helm/Kustomize to manage OAM yamls
```

Comment:
Maybe also add a doc on how to use CNAB to package a OAM defined application.

/cc @technosophos

Reply:
yeah, this could leave to another PR

Reply:
Let's merge this one (parametering), we could leave CNAB part for packaging.